docs/deploying-to-azure.md

```diff
@@ -40,13 +40,18 @@ LOCATION="eastus2"
 # Container Apps
 LOG_ANALYTICS_WORKSPACE="super-heroes-logs"
 CONTAINERAPPS_ENVIRONMENT="super-heroes-env"
-# Kafka
-EVENTHUB_NAMESPACE="super-heroes-eventhub"
-EVENTHUB_TOPIC="fights"
 # Postgres
 POSTGRES_DB="super-heroes-db"
```

Comment:
Should add something in here (like the user's username) so that its unique.

Reply:
I've changed it. Now the each microservice is using its own database. Heroes has a heroes_db and Villains a villains_db database

Reply:
But even still, doesn't the db name need to be unique within the region? If you have one and I have one in the same region, they'd still need to be unique?

Reply:
Yes, you're right. I've added whoami to both DB names